docs/source/models.rst

```diff
@@ -1113,10 +1113,10 @@ Keras (``keras``)
 
 The ``keras`` model flavor enables logging and loading Keras models. It is available in both Python
 and R clients. In R, you can save or log the model using
-``mlflow_save_model`` and ``mlflow_log_model``.
+`mlflow_save_model <R-api.rst#mlflow-save-model>`__ and `mlflow_log_model <R-api.rst#mlflow-log-model>`__.
```

Comment:
Why is this being reverted?

Reply:
I think it may not be update to latest, just pushed a fix